actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -1,651 +0,0 @@
-  #       david = login(:david)
-  #       # User guest logs in
-  #       guest = login(:guest)
-  #
-  #       # Both are now available in different sessions
-  #       assert_equal 'Welcome david!', david.flash[:notice]
-  #       assert_equal 'Welcome guest!', guest.flash[:notice]
-  #
-  #       # User david can browse site
-  #       david.browses_site
-  #       # User guest can browse site as well
-  #       guest.browses_site
-  #
-  #       # Continue with other assertions
-  #     end
-  #
-  #     private
-  #
-  #       module CustomDsl
-  #         def browses_site
-  #           get "/products/all"
-  #           assert_response :success
-  #           assert_select 'h1', 'Products'
-  #         end
-  #       end
-  #
-  #       def login(user)
-  #         open_session do |sess|
-  #           sess.extend(CustomDsl)
-  #           u = users(user)
-  #           sess.https!
-  #           sess.post "/login", params: { username: u.username, password: u.password }
-  #           assert_equal '/welcome', sess.path
-  #           sess.https!(false)
-  #         end
-  #       end
-  #   end
-  #
-  # See the {request helpers documentation}[rdoc-ref:ActionDispatch::Integration::RequestHelpers] for help on how to
-  # use +get+, etc.
-  #
-  # === Changing the request encoding
-  #
-  # You can also test your JSON API easily by setting what the request should
-  # be encoded as:
-  #
-  #   require "test_helper"
-  #
-  #   class ApiTest < ActionDispatch::IntegrationTest
-  #     test "creates articles" do
-  #       assert_difference -> { Article.count } do
-  #         post articles_path, params: { article: { title: "Ahoy!" } }, as: :json
-  #       end
-  #
-  #       assert_response :success
-  #       assert_equal({ id: Article.last.id, title: "Ahoy!" }, response.parsed_body)
-  #     end
-  #   end
-  #
-  # The +as+ option passes an "application/json" Accept header (thereby setting
-  # the request format to JSON unless overridden), sets the content type to
-  # "application/json" and encodes the parameters as JSON.
-  #
-  # Calling +parsed_body+ on the response parses the response body based on the
-  # last response MIME type.
-  #
-  # Out of the box, only <tt>:json</tt> is supported. But for any custom MIME
-  # types you've registered, you can add your own encoders with:
-  #
-  #   ActionDispatch::IntegrationTest.register_encoder :wibble,
-  #     param_encoder: -> params { params.to_wibble },
-  #     response_parser: -> body { body }
-  #
-  # Where +param_encoder+ defines how the params should be encoded and
-  # +response_parser+ defines how the response body should be parsed through
-  # +parsed_body+.
-  #
-  # Consult the Rails Testing Guide for more.
-
-  class IntegrationTest < ActiveSupport::TestCase
-    include TestProcess::FixtureFile
-
-    module UrlOptions
-      extend ActiveSupport::Concern
-      def url_options
-        integration_session.url_options
-      end
-    end
-
-    module Behavior
```

Comment:
I tend to like consistency. I do not exactly care if things are named `XXXTest` or `XXXTestCase` or `XXX::Test(Case?)` but I'd like them to be consistent 😄